manifest-build-tools/application/tag_change_monitor.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+# Copyright 2016, EMC, Inc.
+
+"""
+This is a command line program that monitor repos tag updates.
+When new tag appears, this script will write a property file contains new tag information.
+So a record file which stored history tags in each line is necessary.
+
+usage:
+    ./on-tools/manifest-build-tools/HWIMO-BUILD.py \
+    on-tools/manifest-build-tools/application/tag_change_monitor.py \
+    --repo rackhd/on-http \
+    --history_file workspace/on-http_tag_history
+    --property_file workspace/property_file
+    --credential CREDS
+
+The required parameters:
+    repo: user_name/repo_name that indicates which repo should be monitoring.
+    history_file: A file stored one history tag in each line. If doesn't exists it will be created.
+    property_file: A file used for downstream job.
```

Comment:
as the **output** of script, the comment is not enough...
if I were you , I may say "property_file:  the output of this script, used for downstream Jenkins jobs. this file will contains the newly added tags (comparing with the $hsitory_file), with those tags' corresponding commit/repo_url   "